receiver/journaldreceiver/README.md

```diff
@@ -98,3 +101,34 @@ which is going to effectively retrieve all entries which matches the following s
   - `_SYSTEMD_UNIT` is `ssh`
   - `_SYSTEMD_UNIT` is `kubelet` and `_UID` is `1000`
 
+## Setup and deployment
+
+The user running the collector must have enough permissions to access the journal; not granting them will lead to issues.
+
+When running in a containerized enviroment, differences in the systemd version running on the host and on the container may prevent access to logs due to different features and configurations (e.g. zstd compression, keyed hash etc).
```

Comment:
This should fix the lint failure: `"enviroment" is a misspelling of "environment"`
```suggestion
When running in a containerized environment, differences in the systemd version running on the host and on the container may prevent access to logs due to different features and configurations (e.g. zstd compression, keyed hash etc).
```